src/components/CardTitle.js

```diff
@@ -3,12 +3,11 @@ import React from "react";
 export default function CardTitle() {
   return (
     <div>
-      <div className=" bg-slate-500">
-        <div className="w-[53%] ml-[12.5%] bg-white rounded-lg">
+      
           <div className="p-4 rounded-lg">
             <span className="text-black text-xl font-bold">Don't Miss</span>
-            <div className="flex flex-row gap-12">
-              <div className="">
+            <div className="flex flex-row gap-12 w-[95%]">
```

Comment:
Don't use this - 95 % there should be tailwind classes to achieve this. Connect with Nivya Peter if you have doubts she had done it before.

Reply:
Oki, I just configured the tailwind theme file